src/components/custom-field-input-text/custom-field-input-text.jsx

```diff
@@ -18,9 +18,11 @@ function getRootClassName(className, error, disabled) {
 }
 
 export default function CustomFieldInputText(props) {
-  const inputRef = props.inputRef || useRef(null);
+  const defaultRef = useRef(null);
```

Comment:
For the curious: this was a bug. The `||` is a conditional and would run the hooks in different order depending on a re-render of the component (and some `key` shenanigans). Very similar to "do not run hooks conditional" then we shouldn't invoke `useRef` conditionally. Instead, we always invoke it and then conditionally choose to use its value.